subprojects/docs/src/docs/userguide/troubleshooting.adoc

```diff
@@ -152,6 +152,42 @@ If you're using link:https://projects.eclipse.org/projects/tools.buildship[Build
 .Refreshing a Gradle project in Eclipse Buildship
 image::troubleshooting-refresh-eclipse.png[]
 
+[[network_connection]]
+=== Troubleshooting daemon connection issues
+
+A wrong network configuration can block connections to the Gradle daemon. A typical manifestation of the problem is when you try running a build and it fails immediately with a similar output.
+
+```
+$ gradle help
+
+Starting a Gradle Daemon, 1 stopped Daemon could not be reused, use --status for details
+
+FAILURE: Build failed with an exception.
+
+* What went wrong:
+A new daemon was started but could not be connected to: pid=DaemonInfo{pid=55913, address=[7fb34c82-1907-4c32-afda-888c9b6e2279 port:42751, addresses:[/127.0.0.1]], state=Busy, ...
+```
+
+This may happen when the connection request comes from an IP address that Gradle labels as external (and rejects due to security concerns). We have observed this scenario in some NAT network setup. The solution is to adjust the network configuration such that local network connections are not routed through external addresses.
```

Comment:
```suggestion
We have observed this can occur when network address translation (NAT) masquerade is used. When NAT masquerade is enabled, connections that should be considered local to the machine are masked to appear from external IP addresses. Gradle refuses to connect to any external IP address as a security precaution.

The solution to this problem is to adjust your network configuration such that local connections are not modified to appear as from external addresses. 
```